### DISCUSSION TITLE
Error when i try to open project properties

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#16985] [General]
![image](https://github.com/unoplatform/uno/assets/90481654/3440bfc3-28c0-475a-9d06-fd5a5446107d)
![image](https://github.com/unoplatform/uno/assets/90481654/ce927621-a522-43bc-87d3-c5f200e0aa1e)

After creating new blank project with material, toolkit, and the android and windows platform, I get this error when I try to open the project properties. This happened on both Visual studio Version 17.11.0 Preview 1.1 and Version 17.9.6. I would expect that you could open this page and change  properties from there if you want to. Is there something you're supposed to add to get access to this page?

**Comment by jeromelaban** [comment]
Thanks for the report, see: https://github.com/unoplatform/uno/issues/16490#issuecomment-2145144906 for a workaround.
